backend/de.metas.purchasecandidate.base/src/main/java/de/metas/purchasecandidate/async/C_PurchaseCandidates_GeneratePurchaseOrders.java

```diff
@@ -59,18 +59,23 @@
 
 /**
  * Aggregates enqueued {@link PurchaseCandidate}s and generates purchase orders.
- *
+ * <p>
  * Also, {@link #enqueue(Collection)} method is used to enqueue purchase candidates.
  *
  * @author metas-dev <dev@metasfresh.com>
- *
  */
 public class C_PurchaseCandidates_GeneratePurchaseOrders extends WorkpackageProcessorAdapter
 {
+	public static final String DOC_TYPE_ID = "createAsRequisition";
```

Comment:
```suggestion
	public static final String DOC_TYPE_ID = "C_Doctype_Target_ID";
```

..or maybe `orderDoctTypeId`